.gitignore

```diff
@@ -5,8 +5,8 @@
 build/
 coverage/*
 courtney/
-.idea/
```

Comment:
re-add the `.idea/`, for those using Goland, definitely a must.

Reply:
Its duplicated

Reply:
Please look at `.gitignore:2`

Reply:
Thanks!